app/src/main/java/com/onboarding/calculator/mvp/view/CalculatorView.java

```diff
@@ -28,19 +30,28 @@ public void showError() {
         binding.textViewResult.setText(R.string.calculator_error);
     }
 
+    @Override
+    public void showOperator(String value) {
+        binding.textViewResult.setText(value);
+    }
+
     @Override
     public void showDeleteAllMessage() {
         Context context = getContext();
         if (context != null) {
-            Toast.makeText(context, context.getString(R.string.operation_deleted), Toast.LENGTH_LONG).show();
+            Toast.makeText(context, context.getString(R.string.operation_deleted), Toast.LENGTH_SHORT).show();
         }
     }
 
     @Override
     public void showDeleteMessage() {
         Context context = getContext();
         if (context != null) {
-            Toast.makeText(context, context.getString(R.string.value_deleted), Toast.LENGTH_LONG).show();
+            Toast.makeText(context, context.getString(R.string.value_deleted), Toast.LENGTH_SHORT).show();
         }
     }
+
+    public void reset() {
```

Comment:
pls change the name of the method to resetVisor or resetResultVisor or else

Reply:
Solved!